data-2020.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: End of Term 2020 Dataset
+permalink: /data/data-2020/
+---
+
+# End of Term 2020 Dataset
+
+The End of Term 2020 Dataset represents data collected by three collecting institutions. These institutions were the Internet Archive (IA) and the University of North Texas Libraries (UNT). The data is part of the initiative called the End of Term Presidential Web Archive.  
```

Comment:
So, is it just two collecting institutions then :D?

Reply:
I was never very good with math.